community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -341,6 +341,17 @@
     @Description( "Maximum number of history files for the debug log." )
     public static final Setting<Integer> store_internal_log_max_archives = setting("dbms.logs.debug.rotation.keep_number", INTEGER, "7", min(1) );
 
+    @Description( "Configures the general policy for when check-points should occur. The default policy is to " +
+                  "check-point 'periodically', as specified by the 'dbms.checkpoint.interval.tx' and " +
+                  "'dbms.checkpoint.interval.time' settings. " +
+                  "The Neo4j Enterprise Edition provides two alternative policies: " +
+                  "The first is to check-point 'continuously', which will ignore those settings and run the " +
+                  "check-point process all the time. " +
+                  "The second is the 'volumetric' policy, which makes a best-effort at check-pointing often enough " +
+                  "so that the database doesn't get too far behind on deleting old transaction logs in accordance " +
+                  "with the 'dbms.tx_log.rotation.retention_policy' setting." )
+    public static final Setting<String> check_point_policy = setting( "dbms.checkpoint", STRING, "periodically" );
```

Comment:
They are aliases.

Reply:
they are aliases in threshold classes but they should be aligned here in a property description - since those are the values that user will gonna use. You need to be insane to go to threshold classes to get all the possible values that you can use